### DISCUSSION TITLE
Context constructor

### DISCUSSION THREAD

**Discussion by AliKaanT** [honojs/hono#910] [Q&A] [answered]
Hi. I need an constructor of Context. I need to create my own context object for testing my controller functions. Please help me

**Comment by yusukebe** [comment, marked as answer]
The constructor is here:

https://github.com/honojs/hono/blob/2c5b9891bd1f25205f242bf22651d9fa061abdb1/src/context.ts#L56

But basically, the user should not use `new` to create `Context` directly.

**Reply by joshamaju** [reply]
But how do you pass your custom context to the Hono app?
